recipe/__init__.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+from __future__ import print_function, absolute_import
+
+__version__ = "0.11.7"
```

Comment:
This is replaced by this [`sed` call]( https://github.com/conda-forge/ruamel_yaml-feedstock/pull/3/commits/7dd5b2675165f9197fae044eaa82333b77e1e3ae#diff-cae85152aa5c51d0649ab250c65a640eR28 ).